package.json

```diff
@@ -62,7 +63,11 @@
     "@types/rimraf": "^2.0.2",
     "@types/vinyl": "^2.0.2",
     "@types/webpack": "^4.4.19",
+    "exports-loader": "^1.1.1",
```

Comment:
Are those still needed? If yes, shouldn't they be `dependencies`?

Reply:
yes they are used in the index.ts file and correct, they should be added to the `dependencies`. I guess it worked for me like this because i'm running it locally. I'll move them.